doc/version_history.rst

```diff
@@ -2,6 +2,12 @@
 Version History
 ===============
 
+v0.9.2
```

Comment:
I think you might want to make this v0.10.0, because there's an API breakage here.

Reply:
Fixed in the commit: [4e31c7f](https://github.com/lsst-ts/ts_m2/pull/52/commits/4e31c7fad1a6433f70fbcb3bf660d19b505fa35f). Thanks!